app.py

```diff
@@ -142,7 +143,7 @@ def convert_reference_json(reference_json, data_object):
     # u'url': [u'gs://topmed-irc-share/genomes/NWD145710.b38.irc.v1.cram',
     # u's3://nih-nhlbi-datacommons/NWD145710.b38.irc.v1.cram']}
     data_object['size'] = reference_json['size']
-    data_object['urls'] = map(lambda x: {'url': x}, reference_json['url'])
+    data_object['urls'] = [{'url': x} for x in reference_json['url']]
```

Comment:
I like that you turned the lambdas into list comprehensions - those are also somewhat cryptic, but a breeze compared to lambdas (my personal opinion).